drone.yml

```diff
@@ -0,0 +1,45 @@
+---
+kind: pipeline
+type: docker
+name: nim-on-arm64
+
+platform:
+  os: linux
+  arch: arm64
+
+clone:
+  depth: 1
+
+steps:
+- name: runci
+  image: ghcr.io/alaviss/nim-ci:sha-1d368dc@sha256:b03ad81192e7e14dec4471b63a3d98c78ceada2e675d9075151193d972814b26
```

Comment:
can this be hosted in nim-lang/nimdockers (or some other repo), along with a Dockerfile so that it's reproducible from sources?

Reply:
It's already hosted at alaviss/nim-ci. I can move it to nim-lang, sure.